src/view/global.styl

```diff
@@ -23,6 +23,16 @@ code {
   font-family: monospace;
 }
 
+.u-toolTipMinimumSize {
+  min-width:  120*$px !important;
+}
+
+// This provides enough padding to the right of the 'learn more' link on events
+// to allow the error toolTip to dispaly on the right side of the text field
+.u-gapLinkRight {
+  margin-right: $gap*7 !important;
+}
```

Comment:
This class is for a very specific scenario:

* The link text is "Learn more". If it were a different link, the link may be shorter or longer, in which case this class may not be applicable.
* The link is to the right of a fully expanded textfield.

But the class name makes it seem usable in a broader context, which is probably not great. Here's my take. There are several ways to slice this, but I recommend creating a react component called `selectorsLearnMoreLink.js` that renders this:

```
<Link
      className="SelectorsLearnMoreLink u-gapLeft"
      href="https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Selectors"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn more
    </Link>
```

Create a `selectorsLearnMoreLink.styl` file with this:

```
.SelectorsLearnMoreLink {
  margin-right: 56*$px;
}
```

then use that component in the two views that need it.

Reply:
I would recommend doing something similar with `u-toolTipMinimumSize`. Get rid of the utility class and create a `tooltipPlaceholder.jsx` component that renders:

```
<div className="TooltipPlaceholder"/>
```

And a `tooltipPlaceholder.styl` with this:

```
.TooltipPlaceholder {
  width: 120*$px;
}
```

Then just render `<TooltipPlaceholder/>` wherever you need the placeholder.

Reply:
Great idea,  the `selectorLearnMoreLink.jsx` as well as a `tooltipPlaceholder.jsx` components have been created and are working, however you may see something in the way I implemented it,  I'm a bit new to the react world, there are no lint errors.   Also I made the name singular, "selector" as apposed to plural "selectors" to be consistent with the other components.